### PR TITLE
Added HomePage

### PR DESCRIPTION
**Pull request by Navansh**
## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review
- HomePage added

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/pages/Home/sections/Home/Brands.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        <h2 className='text-[32px] md:text-[42px] font-bold lg:w-[100%] w-[324px]  mx-auto px-1 md:w-[516px]  text-[#183244] text-center md:mx-auto   Quicksand'>Trusted by impactful <br className='hidden lg:flex' /> nonprofits worldwide</h2>
+      
+
+        <div className=' absolute bg-[#ffff] h-[50%] w-[35%] z-[8] top-[56%] translate-y-[-50%] left-[-15%] blur-[70px]   rounded-full'></div>
+
+        <div className='brands_container  whitespace-nowrap overflow-x-auto '>
+           
+            <div className='inline-block md:scale-[0]   whitespace-nowrap brands'>
+                <div className="brand-img-box lg:w-32 lg:ml-8 md:ml-3 ml-2  lg:h-28 md:h-24 md:w-28 w-24 h-20  md:scale-[0.8] lg:scale-[1] scale-[.85]  inline-block"><img src="/brand_1.svg" className='w-full h-full object-cover object-center' alt="brand_1" /></div>
+                <div className="brand-img-box lg:w-32 lg:ml-8 md:ml-3 ml-2  lg:h-28 md:h-24 md:w-28 w-24 h-20  md:scale-[0.8] lg:scale-[1] scale-[.85]  inline-block"><img src="/brand_2.svg" className='w-full h-full object-cover object-center' alt="brand_2" /></div>
```
Comment (SovereignAndrey) [inline src/pages/Home/sections/Home/Brands.jsx:16]: Have you run `yarn format && yarn lint` on these new files yet? These appear to need formating.

File: src/pages/Home/sections/Home/image/Ellipse 32.png
Comment (SovereignAndrey) [inline src/pages/Home/sections/Home/image/Ellipse 32.png:1]: These `Home /image/` pngs could be moved to a `landing` folder under assets.

File: public/BG.jpg
Comment (SovereignAndrey) [inline public/BG.jpg:1]: Most of these public images should probably live in `/src/assets` as we don't need them publically accessible in general (unlike some of our logos and branding images).

**Review by SovereignAndrey — CHANGES_REQUESTED** [review]
Please see my comments below @Navansh on some of the initial things to get this closer to merge ready. Thank you!

**Review by SovereignAndrey — COMMENTED** [review]
Please move all these new landing page JSX components under a `landing` folder.

File: src/pages/Home/styles/BlogPage.css
Comment (SovereignAndrey) [inline src/pages/Home/styles/BlogPage.css:1]: Likewise, all these new styles need to be grouped together under a `landing` folder. Will help with our eventual refactoring to TS that we'll need to perform.

File: src/App/App.tsx [hunk 2]
```diff
@@ -118,6 +120,7 @@ export default function App() {
           path="*"
           element={<Navigate replace to={appRoutes.marketplace} />}
         />
+        <Route path="*" element={<Navigate replace to={appRoutes.home} />} />
```
Comment (SovereignAndrey) [inline src/App/App.tsx:123]: No need to add this, just edit the existing route directly above this one, replacing marketplace with the landing page: https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2894/files#diff-1deef93890e359e3ea9cccd8b9fb992a1008c1f21b9eee105a1a364afae4f170R119-R122

File: src/pages/Home/components/TestimonialCard.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        {/* <img src={props.reviewer_logo} alt="logo" className='h-[80px] w-[80px] absolute top-[-30px] left-[35px] rounded-xl object-cover object-center  bg-white  bg-slate-500'/> */}
+        <div className=" w-[130px] h-[130px]   absolute top-[-10%] left-[20px] ">
+        <img src={props.reviewer_logo} alt="logo" className='w-full h-full object-cover '/> 
+        </div>
+        <p className='lg:text-[18px] md:text-[18px] text-[17px] h-[270px] mt-8 text-ellipsis overflow-x-hidden font-normal w-full break-words  opacity-[.8]'>{props.review || "We are in love with the easy process Better Giving has created to start an endowment. We always focused on recurring giving with donors. This takes it to the next level."}</p>
+        <span>
+            <p className='text-lg opacity-90 font-bold'>{props.reviewer || "Amber Olive"}</p>
+            <p className='lg:text-lg md:text-lg text-[16px] font-medium opacity-75'>{props.reviewer_profession || "Associate director, Australia Aid"}</p>
```
Comment (SovereignAndrey) [inline src/pages/Home/components/TestimonialCard.jsx:17]: Plese don't hardcode dummy data as fallback. 
```suggestion
            <p className='lg:text-lg md:text-lg text-[16px] font-medium opacity-75'>{props.reviewer_profession ?? ""}</p>
```

File: src/pages/Home/hooks/useGetTestimonials.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+
+export default function useGetTestimonials() {
+    const [testimonials, setTestimonials] = useState([
```
Comment (SovereignAndrey) [inline src/pages/Home/hooks/useGetTestimonials.js:6]: All constants, like these and others, should be moved to `srs/constants` file in imported here, not placed in the page itself.

File: src/pages/Home/sections/Home/Blogs.jsx [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+                        return ( <SwiperSlide key={idx} className='w-[300px]' >
+                         <BlogCard blog={ele} imgUrl={thumbBlog[idx]} />
+                       </SwiperSlide>)
+                        // <BlogCard blog={ele} />
+                    })}
+                    {/* <SwiperSlide></SwiperSlide> */}
+                </Swiper>
+                {/* </div> */}
+                
+                {/* yh swipper ka right button ko hide krna ka code hai style={{ display: showNextButton ? "block" : "none" }} */}
```
Comment (ap-justin) [inline src/pages/Home/sections/Home/Blogs.jsx:165]: Kindly remove any comments that might not have future use - if to be used in the future kindly prepend with `//Future: ...comment`

**Review by ap-justin — CHANGES_REQUESTED** [review]

File: src/App/App.tsx [hunk 6]
```diff
@@ -28,6 +28,7 @@ const StripePaymentStatus = lazy(() => import("pages/StripePaymentStatus"));
 const Widget = lazy(() => import("pages/Widget"));
 const BlogPosts = lazy(() => import("pages/Blog/Posts"));
 const BlogPost = lazy(() => import("pages/Blog/Post"));
+const Home = lazy(() => import("pages/Home/Homepage/Home"));
```
Comment (ap-justin) [inline src/App/App.tsx:31]:
```suggestion
const Home = lazy(() => import("pages/Home/Home"));
```
`HomePage` is extra folder, 

could be 
```suggestion
const Home = lazy(() => import("pages/Home"));
```

if you add `Home/index.ts` which handles export
```
//Home/index.ts
export {default} from "./Home.tsx"
```

**Review by ap-justin — CHANGES_REQUESTED** [review]

File: src/pages/Home/components/AnimatedSVG.jsx [hunk 7]
```diff
@@ -0,0 +1,637 @@
...
+import { ScrollTrigger } from "gsap/ScrollTrigger";
+import { MotionPathPlugin } from "gsap/MotionPathPlugin";
+import { EasePack } from "gsap/EasePack";
+import { CustomEase } from "gsap/CustomEase";
+import "../styles/svgAnimation.css";
+gsap.registerPlugin(EasePack, CustomEase, ScrollTrigger, MotionPathPlugin);
+
+gsap.registerPlugin(ScrollTrigger, MotionPathPlugin);
+
+const AnimatedSVG = () => {
```
Comment (ap-justin) [inline src/pages/Home/components/AnimatedSVG.jsx:14]: at certain page sizes, this animated SVG breaks layout
image

**Review by ap-justin — CHANGES_REQUESTED** [review]

File: src/pages/Home/App.css [hunk 8]
```diff
@@ -0,0 +1,379 @@
...
+/* background: url(<path-to-image>), lightgray 0% 0% / 50px 50px repeat, linear-gradient(180deg, rgba(255, 255, 255, 0.50) 0%, rgba(251, 241, 235, 0.96) 87.19%, #FFF 94.34%); */
+
+.DM_Sans{
+    font-family: "DM Sans";
+}
+.vag{
+    font-family: vag;
+    /* font-weight: 300; */
+}
+.Quicksand{
```
Comment (ap-justin) [inline src/pages/Home/App.css:31]: codebase already implements quicksand with `font-heading` class, and can be

File: src/pages/Home/App.css [hunk 9]
```diff
@@ -0,0 +1,379 @@
...
+    background: linear-gradient(180deg, rgba(255, 255, 255, 0.50) 0%, rgba(251, 241, 235, 0.96) 87.19%, #FFF 94.34%);
+ 
+    background-blend-mode: screen, normal;
+    
+    overflow: hidden;
+    background-color: #fff;
+}
+/* background: url(<path-to-image>), lightgray 0% 0% / 50px 50px repeat, linear-gradient(180deg, rgba(255, 255, 255, 0.50) 0%, rgba(251, 241, 235, 0.96) 87.19%, #FFF 94.34%); */
+
+.DM_Sans{
```
Comment (ap-justin) [inline src/pages/Home/App.css:24]: codebase already implements dm sans via `font-body` class

File: src/pages/Home/App.css [hunk 10]
```diff
@@ -0,0 +1,379 @@
...
+}
+.vag{
+    font-family: vag;
+    /* font-weight: 300; */
+}
+.Quicksand{
+    font-family: "Quicksand";
+}
+
+.Gochi_Hand{
```
Comment (ap-justin) [inline src/pages/Home/App.css:35]: `.Gochi_Hand` could be defined in `tailwind.config.js`
```
fontFamily: {
        heading: ["Quicksand", "serif"],
        body: ["DM Sans", "sans-serif"],
        gochi: ["Gochi Hand", "serif"]
      },
```
and be used as 
```
<span className="font-gochi" ....
```

File: src/pages/Home/App.css [hunk 11]
```diff
@@ -0,0 +1,379 @@
...
+.shadow {
+    box-shadow: 0px 12px 36px 0px rgba(0, 0, 0, 0.06);
+}
+
+.shadow_1 {
+    box-shadow: 0px 12px 6px 0px rgba(80, 79, 79, 0.06);
+}
+
+/*  Hero.jsx  */
+.bg_blur {
```
Comment (ap-justin) [inline src/pages/Home/App.css:48]: this custom class can be removed 
```
<div className=".bg_blur...
```
to
```
<div className="absolute top-0 left-0 w-full h-[60%] -z-1 bg-[#fdfdfd] blur-[400px] />
```

File: src/pages/Home/App.css [hunk 12]
```diff
@@ -0,0 +1,379 @@
...
+    from{
+        transform: translateX(0);
+    }
+    to{
+        transform: translateX(-100%);  
+    }
+}
+
+
+.brands {
```
Comment (ap-justin) [inline src/pages/Home/App.css:87]: this class is unused, and can be removed - kindly look for other instances of unused classes

**Review by ap-justin — CHANGES_REQUESTED** [review]

File: package.json [hunk 13]
```diff
@@ -62,6 +65,7 @@
     "react-icons": "5.0.1",
     "react-redux": "9.1.0",
     "react-router-dom": "6.22.2",
+    "swiper": "^11.1.1",
```
Comment (ap-justin) [inline package.json:68]:
```suggestion
    "swiper": "11.1.1",
```
kindly use exact version for consistency with the rest of the dependencies

**Review by ap-justin — CHANGES_REQUESTED** [review]

File: src/pages/Home/components/AnimatedSVG.jsx (hunk 7 above)
Comment (Navansh) [reply src/pages/Home/components/AnimatedSVG.jsx:14]: I'm not getting this fully, can you please explain more ?

File: src/pages/Home/components/AdminNavbar.jsx [hunk 14]
```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from 'react'
+import { PageContext } from '../state/PageState'
+import { UserContext } from '../state/UserState'
+
+const AdminNavbar = () => {
```
Comment (ap-justin) [inline src/pages/Home/components/AdminNavbar.jsx:5]: unused component, kindly remove together with other unused ones

**Review by ap-justin — CHANGES_REQUESTED** [review]

File: src/pages/Home/components/AnimatedSVG.jsx (hunk 7 above)
Comment (ap-justin) [reply src/pages/Home/components/AnimatedSVG.jsx:14]: > at certain page sizes,

e.g. ipad pro `1024px` wide, page loads like this (with overflows)

image

Comment (Navansh) [reply src/pages/Home/components/AnimatedSVG.jsx:14]: Actually this is difference between the breakout considered for the better.giving website and our landing page design, so there exists conflicting situation in this 1000px range

File: src/pages/Home/App.css (hunk 12 above)
Comment (Navansh) [reply src/pages/Home/App.css:87]: This class is used inside brands section

Comment (Navansh) [reply src/pages/Home/App.css:87]: image
****

File: src/pages/Home/components/AdminNavbar.jsx (hunk 14 above)
Comment (Navansh) [reply src/pages/Home/components/AdminNavbar.jsx:5]: image
Its used here

File: src/pages/Home/components/AnimatedSVG.jsx (hunk 7 above)
Comment (SovereignAndrey) [reply src/pages/Home/components/AnimatedSVG.jsx:14]: @Navansh We can address those break point discrepancies later internally. If it look OK for now at a general desktop & mobile size, we can let this slide.  
cc: @ap-justin

**Pull request merged**